test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -289,8 +292,94 @@ static void prepareDatabase(void) {
   connection->insert(SUBSCRIBECONTEXT_COLL, sub2);
   connection->insert(SUBSCRIBECONTEXT_COLL, sub3);
 
+  /* Given that preparation including csubs, we have to init cache */
+  if (initializeCache == true)
```

Comment:
Is this checking really needed? Each TEST() is independent, the DB is cleaned at the beginning of prepareDatabase() so why don't just invoking subscriptionCacheInit("")?